### PR TITLE
add shebang line to script in tools

### PR DESCRIPTION
**Pull request by fabbox**
This update fix https://github.com/pdfminer/pdfminer.six/issues/405. 

It enables to run the scripts in tools (especially pdf2txt.py and dumppdf.py) directly from terminal.

### CODE REVIEW

**Comment by loqs** [comment]
Should it not be python3 as the python 2 no longer supported in line with https://www.python.org/dev/peps/pep-0394/ ?

**Comment by fabbox** [comment]
As python2 is now "deprecated" since January 2020, I didn't think about it. I must admit that according to https://www.python.org/dev/peps/pep-0394/#future-changes-to-this-recommendation, authors says the PEP394 should be update if the support for python2 is completly drop. On the other hand, most of scripts already containing the shebang line used the "python" and not "python3", so I really don't know.

File: tools/dumppdf.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```
Comment (pietermarsman) [inline tools/dumppdf.py:1]: Let's use python3 as suggested by pep394 (thanks for the reference :) )

**Comment by eli-schwartz** [comment]
@fabbox the Python PEP *requirement* is that for programs, such as this, which are incompatible with python2 and only work on python3, the shebang should specifically reference python3.

It is wrong to use `#!/usr/bin/env python` anywhere in this project, ever since support for python2 was dropped in 3502dc9f3bc81de721fccde782da844ae8239a0a. The current PR as-is will break Debian, since IIRC debian decided to completely remove the /usr/bin/python command rather than make it a symlink to python3.

**Comment by eli-schwartz** [comment]
Why did you close this PR, abandon the review, and open a new PR with the exact same content?

**Comment by fabbox** [comment]
I close this pull request and creat a new one with the proposition (with python3).  
Sorry for the long delay between the two propositions.. I though it was the way to do. Should I be able to edit this PR ?

**Comment by eli-schwartz** [comment]
Sure, this PR even updated when you pushed additional commits, then a minute later you closed it.

**Comment by fabbox** [comment]
OK, I think I miss something in the PR workflow. I didn't find how to edit the PR. I will close the new one and try to update this one.

**Comment by fabbox** [comment]
Sorry to all, my bad here, I was thinking it do not accept my PR but it was updating it... 
I completly agreed with the proposed change so I updated it with python3 everywhere the shebang line appeared.

File: tools/dumppdf.py (hunk 1 above)
Comment (fabbox) [reply tools/dumppdf.py]: Yes, done !

**Comment by pietermarsman** [comment]
@fabbox Thanks!

**Comment by estshorter** [comment]
I hope this fix will be released soon.

I'm a maintainer of a [pdfminer.six-feedstock repositry](https://github.com/conda-forge/pdfminer.six-feedstock) which enables installation of pdfminer.six via conda.
The latest release (20200402) breaks [tests ](https://github.com/conda-forge/pdfminer.six-feedstock/blob/master/recipe/meta.yaml#L27-L34)because of no shebang line, and I cannot upgrade version on conda...

**Comment by pietermarsman** [comment]
@estshorter done!

Let me know if it works.

**Comment by estshorter** [comment]
Thanks @pietermarsman !
Now we can get the latest pdfminer.six via conda.

**Pull request merged**